docs/example-scenario/apps/sap-dev-test.md

```diff
@@ -0,0 +1,109 @@
+---
+title: SAP for a dev/test environment
+description: SAP scenario for a dev/test environment
+author: AndrewDibbins
+ms.date: 7/11/18
+---
+
+# SAP for a dev/test workload
+
+This architectural design shows a set of proven practices for running SAP NetWeaver in a Windows or Linux environment on Azure. The database is AnyDB, the SAP term for any supported DBMS (that isn't SAP HANA). This architecture is deployed with specific virtual machine (VM) sizes that can be changed to accommodate your organization's needs.
```

Comment:
Lets talk about dev/test in here  maybe:

This example provides guidance for how to run a dev/test implementation of SAP NetWeaver in a Windows or Linux environment on Azure. The database used is AnyDB, the SAP term for any supported DBMS (that isn't SAP HANA). This architecture is deployed with specific virtual machine (VM) sizes that can be changed to accommodate your organization's needs.

Reply:
Also mention what makes it dev/test vs production - single VM etc

Reply:
Can we also change the name slightly?  "SAP for a dev/test workload" sounds a bit awkward.  How about "SAP for dev/test workloads"?

As far as the language, how about changing the last line to:

Because this architecture is designed for non-production environments, it's deployed with just a single virtual machine (VM) and it's size can be changed to accommodate your organization's needs.